openapi.yaml

```diff
@@ -186,6 +185,55 @@ paths:
           description: Internal Server Error
           schema:
             $ref: '#/definitions/ErrorResult'
+  /onbase/documents:
+    post:
+      tags:
+        - onBase
+        - documents
+      description: Insert document data
+      operationId: postDocument
+      parameters:
+        - in: body
+          name: postDocumentBody
+          required: true
+          schema:
+            properties:
+              data:
+                type: object
+                properties:
+                  type:
+                    $ref: '#/definitions/DocumentsType'
+                  attributes:
+                    $ref: '#/definitions/PostDocumentAttributes'
+                    required:
+                      - documentId
+                      - fieldName
+                      - indexKey
+                      - docTypeNumber
+                      - instance
```

Comment:
These fields aren't showing as required on the generated view because `PostDocumentAttributes` resolves to  `properties {...}`. The `required` list needs to be inside of `properties`.

Reply:
Good catch, fixed.